README.md

```diff
@@ -26,6 +26,10 @@ the following code should be used to pass it to the action:
 
 _Optional_ The name you want to use in the Docker registry. The name can only contain lowercase letters, numbers, hyphens ("-"), and underscores ("_").
 
+### `exiting-image`
```

Comment:
Nit: I guess this should be `existing-image`.

Reply:
Fixed 👍